src/plugins/mf_priority.cpp

```diff
@@ -895,6 +898,38 @@ static int depend_cb (flux_plugin_t *p,
         return -1;
     }
 
+    if (queue != NULL) {
+        // safely assign "queue" to an std::string
+        std::string value (queue);
+        // fetch max number of running jobs in this queue
+        int queue_max_run_jobs = max_run_jobs_per_queue (queues, queue);
+        if (queue_max_run_jobs < 0) {
+            // can't find a max_run_jobs limit for this queue because it
+            // might not be defined in the flux-accounting DB, so just set
+            // it to a large number
+            queue_max_run_jobs = std::numeric_limits<int>::max ();
+        }
+
+        // look up the association's current number of running jobs;
+        // if queue cannot be found, an entry in the Association object will be
+        // initialized with a current running jobs count of 0
+        int assoc_cur_run_jobs = b->queue_usage[queue];
+        if (assoc_cur_run_jobs == queue_max_run_jobs) {
```

Comment:
Could be `>=` here right? Shouldn't hurt? or help I guess

Reply:
Oh yeah, good suggestion, you're right. Shouldn't hurt!